db/schema.rb

```diff
@@ -10,9 +10,30 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_09_09_083144) do
+ActiveRecord::Schema.define(version: 2020_09_22_071405) do
 
-  create_table "comments", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci", force: :cascade do |t|
```

Comment:
this shouldnt be pushed, discuss with mujeeb for gitignore.

Reply:
ok